cli/src/commit_templater.rs

```diff
@@ -1225,6 +1225,18 @@ fn builtin_commit_or_change_id_methods<'repo>(
     // Not using maplit::hashmap!{} or custom declarative macro here because
     // code completion inside macro is quite restricted.
     let mut map = CommitTemplateBuildMethodFnMap::<CommitOrChangeId>::new();
+    map.insert("hex", |_language, _build_ctx, self_property, function| {
```

Comment:
nit: might be better to call it a `plain_hex()`, `normal_hex()`, etc.? The reverse hex is the canonical hex format for change id.

Can you also update docs/templates.md?